### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by VishnuVNairCodes**

### CODE REVIEW

File: src/components/NavBar.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+.navigation-bar .textbox {
+  outline: none;
+  caret-color: var(--primary);
+  color: var(--primary);
+}
+
+.navigation-login > .btn {
+  /* font-size: 1rem; */
```
Comment (simran61) [inline src/components/NavBar.css:28]: Can remove comments if not needed

File: src/pages/HomePage/components/ProductDisplay3.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import "./ProductDisplay3.css";
+
+const ProductDisplay3 = () => {
+  return (
+    <>
+      <div className="display-3">
+        <div className="display-3-sub-container">
+          <img
+            src={require("../../../assets/baby-mom.jpg")}
```
Comment (simran61) [inline src/pages/HomePage/components/ProductDisplay3.js:9]: Nice way of using "require" for images instead of directly importing them.

**Review by simran61 — APPROVED** [review]
Looks good

File: src/components/NavBar.css (hunk 1 above)
Comment (VishnuVNairCodes) [reply src/components/NavBar.css:28]: Thank you for pointing this. Will remove

File: src/pages/HomePage/components/ProductDisplay3.js (hunk 2 above)
Comment (VishnuVNairCodes) [reply src/pages/HomePage/components/ProductDisplay3.js:9]: Thank you

**Comment by AswathyVNair** [comment]
Good to merge

**Review by AswathyVNair — APPROVED** [review]

File: src/pages/HomePage.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
+import React from "react";
+
+const HomePage = () => {
+  return (
+    <>
+      <div class="outer-container">
```
Comment (veereshbv04) [inline src/pages/HomePage.js]: replace class with className

Comment (VishnuVNairCodes) [reply src/pages/HomePage.js]: Thank you for pointing this out.

**Pull request merged**